go/registry/api/api.go

```diff
@@ -66,7 +66,7 @@ var (
 
 	// RegisterUnfreezeNodeSignatureContext is the context used for
 	// unfreezing nodes.
-	RegisterUnfreezeNodeSignatureContext = signature.NewContext("oasis-core/registry: unfreeze node")
+	RegisterUnfreezeNodeSignatureContext = signature.NewContext("oasis-core/registry: unfreeze node", signature.WithChainSeparation())
```

Comment:
We don't have a method for actually generating these via CLI, but this would break it if such a thing exists.

Reply:
I've changed this back for now.